templates/docker-container-functionapp.properties.json

```diff
@@ -1,4 +1,7 @@
 {
   "iconName": "functionapp",
-  "requires": [ "azuresubscription"]
+  "requires": [ 

```

Comment:
Given that TaskInputDefinition has a `Required` parameter, "parameters" would be a better property name than "requires". If you want to wait and make that change separately I think that's fine.

Reply:
I'm just going to change it now since I was already thinking about it. I just added the extra parameter but I'll rename property as well. Just doing some testing

Reply:
Ok I updated the server changes I'm making to adjust for this and everything still works 👍